### PR TITLE
Issue #603 ; using sensible defaults for image styles.

### PR DESCRIPTION
**Pull request by serundeputy**
This solves: https://github.com/backdrop/backdrop-issues/issues/603

### CODE REVIEW

File: core/modules/image/image.install [hunk 1]
```diff
@@ -180,8 +180,8 @@ function image_update_1000() {
       array(
         'name' => 'image_scale',
         'data' => array(
-          'width' => 480,
-          'height' => 480,
+          'width' => 1230,
+          'height' => 1230,
```
Comment (quicksketch) [inline core/modules/image/image.install:184]: This should match the suggestion above (800x600). Labels need updating as well.

File: core/modules/image/image.install [hunk 2]
```diff
@@ -159,8 +159,8 @@ function image_update_1000() {
       array(
         'name' => 'image_scale',
         'data' => array(
-          'width' => 220,
-          'height' => 220,
+          'width' => 350,
+          'height' => 350,
```
Comment (quicksketch) [inline core/modules/image/image.install:163]: Height should be 250 to match the JSON file. Label needs updating here too

**Comment by hellomrcat** [comment]
@quicksketch it looks like this was fixed in https://github.com/backdrop/backdrop/pull/896, can this PR be closed?

**Comment by quicksketch** [comment]
Yes, thank you!

**Pull request closed without merging**